conf/db.sql

```diff
@@ -98,25 +98,24 @@ DROP TABLE IF EXISTS `Emails`;
 /*!40101 SET character_set_client = utf8 */;
 CREATE TABLE `Emails` (
   `EmailID` bigint(20) NOT NULL AUTO_INCREMENT,
-  `UserID` bigint(20) NOT NULL DEFAULT '0',
+  `UserID` bigint(20) NOT NULL DEFAULT 0,
   `Prefix` varchar(20) DEFAULT NULL,
   `Word` varchar(20) NOT NULL DEFAULT '',
-  `InitialCount` tinyint(4) NOT NULL DEFAULT '0',
-  `Count` tinyint(4) NOT NULL DEFAULT '0',
+  `InitialCount` tinyint(4) NOT NULL DEFAULT 0,
+  `Count` tinyint(4) NOT NULL DEFAULT 0,
   `TimeAdded` bigint(20) DEFAULT NULL,
   `Sender` varchar(254) DEFAULT NULL,
-  `Address` text,
-  `DeliveryAddress` varchar(255) DEFAULT '',
-  `Hidden` tinyint(4) NOT NULL DEFAULT '0',
+  `Address` text DEFAULT NULL,
+  `Hidden` tinyint(4) NOT NULL DEFAULT 0,
   `Note` varchar(254) DEFAULT NULL,
   `PrivateHash` varchar(50) NOT NULL DEFAULT '',
-  `NumForwarded` bigint(20) NOT NULL DEFAULT '0',
-  `NumDeleted` bigint(20) NOT NULL DEFAULT '0',
+  `NumForwarded` bigint(20) NOT NULL DEFAULT 0,
+  `NumDeleted` bigint(20) NOT NULL DEFAULT 0,
   `ExpireTime` bigint(20) unsigned DEFAULT NULL,
   PRIMARY KEY (`EmailID`),
   KEY `UserID` (`UserID`,`Word`,`Count`),
   KEY `Word` (`Word`)
-) ENGINE=MyISAM AUTO_INCREMENT=8443939 DEFAULT CHARSET=latin1;
+) ENGINE=MyISAM AUTO_INCREMENT=9212076 DEFAULT CHARSET=latin1;
```

Comment:
This new number seems related to the existing dataset size instead of being required to recreate the DB. While it is fine to start the next `EmailID` at 9212076 this change (and some future point, this line) can be ommitted. That said, there is no harm in keeping this change for preserving the changes.

Reply:
(similar comment for other occurences)